package.json

```diff
@@ -86,6 +86,8 @@
     "passport.socketio": "~3.7.0",
     "pg": "~6.1.2",
     "pg-hstore": "~2.3.2",
+    "prom-client": "^12.0.0",
```

Comment:
Is it necessary?

Reply:
> Is it necessary?

This package ```prom-client``` is necessary, due to it is a peer dependency package of ```prometheus-api-metrics```.

https://www.npmjs.com/package/prometheus-api-metrics#goal

The package ```prom-client``` implements the Prometheus metrics interface, but it is not bundled by any web framework.

The package ```prometheus-api-metrics``` bundles Prometheus metrics as the express middleware, and exports Process/API metrics.